packages/extension-manager/src/node/extension-backend-module.ts

```diff
@@ -60,4 +62,6 @@ export default new ContainerModule(bind => {
             return server;
         });
     }).inSingletonScope();
+
+    bind(MetricsContribution).to(ExtensionManagerMetricsContribution);
```

Comment:
i wonder how can we do it optional?

Reply:
You mean contribute only if the metrics extension is present ? 
That's something I wonder about too...

Reply:
Maybe the backend could have a config value (or an env variable for now) where you tell it to collect or not? Defaults to false maybe?

Reply:
But where would we want that variable? Per extension (choose which extensions to disable/enable metrics) or only for the metrics server (collect all or nothing)

Reply:
A third extension would be a solution.

Reply:
Maybe keep the first commit of this PR (the one that adds the contribution point) and rethink the second one? A third extension for getting metrics only for one extension is a bit too much no? Unless we would want it this way for all possible extensions and those that want to provide metrics i.e. And we can't one big extension that checks metrics for all other extensions as that would introduce too many dependencies...

Reply:
My main concern is forcing prometheus on everybody. Could you elaborate what it does?

Reply:
Yes that is something I'm concerned with also. It collects and monitors information about running systems and allows you to graph this information with other services like grafana and whatnot. Useful for people that would want to have several dozens of clients on different hosts I think. Although I agree It doesn't quite make sense for the common user to have this if he's never gonna use that information.

I'll discuss with @marcdumais-work about a possible better solution.

For now if I remove the second commit but keep the first one I think it's okay though no?